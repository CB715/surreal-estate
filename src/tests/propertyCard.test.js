import React from "react";
import { render } from "@testing-library/react";

import PropertyCard from "../components/propertyCard";

describe("Property Card", () => {
  const property = {
    title: "A test property",
    type: "Flat",
    bathrooms: 1,
    bedrooms: 1,
    price: "Â£1500",
    city: "Leeds",
    email: "test@test.com",
  };

  it("renders the property card", () => {
    const { asFragment } = render(
      <PropertyCard
        title={property.title}
        type={property.type}
        bathrooms={property.bathrooms}
        bedrooms={property.bedrooms}
        price={property.price}
        city={property.city}
        email={property.email}
      />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
