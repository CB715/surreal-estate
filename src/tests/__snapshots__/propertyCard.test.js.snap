// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Property Card renders the property card 1`] = `
<DocumentFragment>
  <div>
    <p>
      Property Card
    </p>
  </div>
</DocumentFragment>
`;
